terraform/accounts/verify/clusters/prod/cluster.tf

```diff
@@ -51,6 +51,12 @@ module "gsp-cluster" {
     }
 }
 
+module "hsm" {
+  source       = "../../../../modules/hsm"
```

Comment:
This is very specific to Verify - eIDAS.

I wonder if it would be worth moving the module somewhere else...